src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+
+    public static Color getRandomColor() {
```

Comment:
Please do not use static methods in your solution.
Static methods are generally much harder to maintain and test so should be used only when static is required